schema-lending.graphql

```diff
@@ -110,8 +140,8 @@ enum InterestRateSide {
 }
 
```

Comment:
In addition to InterestRateType and InterestRateSide, there is need for an optional third enum for interest rate id: InterestRateTranche, to support tranched pools (Goldfinch, Bainbridge, tranched, etc) as each tranche has different rates.
```
enum InterestRateTranche {
      SENIOR
      JUNIOR
}
```